docs/pages/application-access/getting-started.mdx

```diff
@@ -23,42 +41,47 @@ Let's connect to Grafana using Teleport in three steps:
 
 - (!docs/pages/includes/tctl.mdx!)
 - A host where you will run the Teleport Application Service.
+- A DNS subdomain for each application.
+   
+   <Admonition type="note" title="Application Access and DNS for Self-Hosted Clusters">
 
-<Tabs>
-<TabItem label="Docker">
-- A Docker installation, which we will use to launch Grafana in a container.
-  Alternatively, if you have another web application you'd like to protect with
-  Teleport, you can use that instead.
-</TabItem>
-<TabItem label="Kubernetes Cluster">
+   (!docs/pages/includes/dns-app-access.mdx!)
 
-(!docs/pages/includes/kubernetes-access/helm-k8s.mdx!)
+   Teleport Cloud automatically provides a subdomain and signed TLS certificate for 
+   your application under your tenant address.
+   </Admonition>
 
-- Administrative rights to deploy within a cluster.
+- Docker or Kubernetes to launch Grafana.
 
-</TabItem>
-</Tabs>
+   <Tabs>
+   <TabItem label="Docker">
+   You must have Docker installed to launch any application in a Docker container. 
+   If you are configuring access to any web application using Teleport, use the 
+   appropriate commands to launch that application in a Docker container.
```

Comment:
Should the Docker container be launched before the user starts this guide? If so, why are we having the user run `docker run` with Grafana below?